### PR TITLE
airdrop changes

### PR DESCRIPTION
**Pull request by 0x-genesys**
- added truffle config
- added function in airdrop

### CODE REVIEW

File: migrations/2_deploy_contracts.js [hunk 1]
```diff
@@ -3,9 +3,10 @@ var AirDrop = artifacts.require("./AirDrop.sol");
 var SpringToken = artifacts.require("./SPRINGToken.sol");
 var Attestation = artifacts.require("./Attestation.sol");
 
-module.exports = function(deployer,network,accounts) {
-    deployer.deploy(SpringToken,1000000).then(function() {
-        return deployer.deploy(AirDrop, SpringToken.address);
+module.exports = function(deployer, network, accounts) {
+    deployer.deploy(SpringToken, 1000000)
+    .then(function() {
+        return deployer.deploy(AirDrop, 0x1d08d05638a2a006707cec99dba40aae3b9d54fc);
```
Comment (vinay035) [inline migrations/2_deploy_contracts.js]: @0x-genesys lets not hard code contract address.

File: package.json [hunk 2]
```diff
@@ -8,7 +8,8 @@
   },
   "dependencies": {
     "solc": "^0.4.25",
-    "truffle": "^4.1.14"
+    "truffle": "^4.1.14",
+    "truffle-hdwallet-provider-privkey": "^0.3.0"
```
Comment (vinay035) [inline package.json:12]: why do we need hd provider ?

File: truffle.js [hunk 3]
```diff
@@ -1,10 +1,20 @@
...
   networks: {
     development: {
       host: "localhost",
       port: 8545,
       gas: 6721975,
       network_id: "*" // Match any network id
+    },
+    ropsten: {
+      provider: function() {
+              return new HDWalletProvider(["87BAA9CC66610DA8466516EFC29A6AFC167113EAEAE907637874A4F9AF01868F"],
```
Comment (vinay035) [inline truffle.js]: lets not put keys here.

**Review by vinay035 — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (0x-genesys) [reply package.json:12]: fo truffle.js. line 13

**Review by vinay035 — APPROVED** [review]

**Pull request merged**